### PR TITLE
Add Script to Support Ads Hosted on Hashtag Labs' Ad Server

### PR DESCRIPTION
**Pull request by ashleyjelks**
This PR adds scripts to the article pages to support externally hosting ads from Hashtag Labs' ad servers. 

[Links to GROW-1202](https://artsyproduct.atlassian.net/browse/GROW-1202)

### CODE REVIEW

File: src/Components/Publishing/Ads/AdScript.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+      case (article.layout === "series" || article.layout === "video") &&
+        this.isSponsored(article):
+        return ArticleAdType.SponsorLanding
+
+      default:
+        return ArticleAdType.Article
+    }
+  }
+  isSponsored(article: ArticleData) {
```
Comment (damassi) [inline src/Components/Publishing/Ads/AdScript.tsx]: #minor but can we add line breaks in between each of these class methods so things don't all run together?

File: src/Components/Publishing/Ads/AdScript.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      sd.NODE_ENV || (process.env && process.env.NODE_ENV) === "production"
+        ? "no"
+        : "yes"
+    const pageType = this.getArticleType(article)
+
+    const script = `
+      <script>
+      htlbid.cmd = htlbid.cmd || [];
+      htlbid.cmd.push(function() {
+        htlbid.setTargeting('is_testing', '${env}'); // output 'yes' on stage, 'no' on prod
```
Comment (damassi) [inline src/Components/Publishing/Ads/AdScript.tsx]: This comment can be moved up above line 45

File: src/Components/Publishing/Ads/AdScript.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        htlbid.setTargeting('page_type', '${pageType}'); // output 'feature', 'article', 'newslanding', 'sponsorlanding', 'sponsorfeature'
```
Comment (damassi) [inline src/Components/Publishing/Ads/AdScript.tsx]: And this one above 52

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 7ada01d1558d7c6ef2471d6a8f958218474a4785

https://deploy-preview-2361--artsy-reaction.netlify.com

File: src/Components/Publishing/Series/__tests__/SeriesAbout.test.tsx [hunk 4]
```diff
@@ -59,6 +59,7 @@ describe("SeriesAbout", () => {
 
     it("Does not render PartnerBlock for an unsponsored series", () => {
       const component = mount(<SeriesAbout article={SeriesArticle} />)
+      console.log(component.debug())
```
Comment (l2succes) [inline src/Components/Publishing/Series/__tests__/SeriesAbout.test.tsx]: stray `console.log`

Comment (ashleyjelks) [reply src/Components/Publishing/Series/__tests__/SeriesAbout.test.tsx]: Woops, good catch!

**Comment by artsyit** [comment]
:rocket: PR was released in v16.7.1 :rocket:

**Pull request merged**